CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+### [3.1.0+portage-3.1.4] - 2023-07-13
```

Comment:
We haven't cut the release yet so we can leave this as a part of the ```Unreleased``` section. Also, there are some details that we need to discuss about the versioning system we are using.